src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```diff
@@ -67,13 +68,13 @@ public function onKernelException(GetResponseForExceptionEvent $event)
         try {
             $response = $event->getKernel()->handle($request, HttpKernelInterface::SUB_REQUEST, true);
         } catch (\Exception $e) {
-            $this->logException($exception, sprintf('Exception thrown when handling an exception (%s: %s)', get_class($e), $e->getMessage()), false);
+            $this->logException($e, sprintf('Exception thrown when handling an exception (%s: %s at %s line %s)', get_class($e), $e->getMessage(), $e->getFile(), $e->getLine()), false);
```

Comment:
Is the parent exception logged properly? In that case, I prefer to see a smaller message or log both exceptions.